myproject/blog/views.py

```diff
@@ -40,11 +43,10 @@ class BlogDetail(generic.DetailView,FormMixin):
     def get_context_data(self, **kwargs: Any):
         context = super().get_context_data(**kwargs)
         post = self.get_object()
-        context['comments'] = post.comments.all().order_by("created_at")
+        context['comments'] = post.comments.all()
```

Comment:
Same for here, attached the screenshot here how many queries are being executed and reduce the repeated queries.

Reply:
![Uploading Screenshot 2024-03-29 at 12.09.30 PM.png…]()